code/__DEFINES/equipment.dm

```diff
@@ -6,6 +6,7 @@
 #define PASSMOB (1<<4)
 #define PASSSMALLSTRUCT (1<<5)
 #define PASSFIRE (1<<6)
+#define PASSXENO (1<<7)
```

Comment:
Why did you add this flag I dont see its use since you can shuffle past anyway

Reply:
Allow to shuffle even in harm intent. Since you are surrounded by spiderling all the time that's better